### DISCUSSION TITLE
Is it possible to copy and past to the VNC console?

### DISCUSSION THREAD

**Discussion by Verhoeckx** [cockpit-project/cockpit-machines#293] [Q&A]
Short question:
Is it possible to copy and past to the VNC console?
Is there a shortcut that I'm missing?

`CTRL (+ SHFT) + C` and `CTRL (+ SHFT) + V` doesn't work.

No big of a deal though.

**Comment by KKoukiou** [comment]
I think this is not implemented in @patternfly/react-console VNC component. We use https://github.com/novnc/noVNC there,  I guess we can look into adding support for clipboard.

**Reply by Verhoeckx** [reply]
Reading the feature list of noVNC, it must be possible. Shall I create an issue (feature request) for this?

**Reply by KKoukiou** [reply]
Sure thanks :) But that needs to reported against https://github.com/patternfly/patternfly-react/issues, which is creating the react component that we consume for the VNC console.

**Reply by Verhoeckx** [reply]
I created an issue here:
https://github.com/patternfly/patternfly-react/issues/6092

**Comment by gnat** [comment]
Would be really nice!
